README.md

```diff
@@ -129,15 +129,24 @@ those? Of course!
 
 The `scenic:view` and `scenic:model` generators accept a `--materialized`
 option for this purpose. When used with the model generator, your model will
-have the following method defined as a convenience to aid in scheduling
+have the following methods defined as conveniences to aid in scheduling
 refreshes:
 
 ```ruby
 def self.refresh
   Scenic.database.refresh_materialized_view(table_name)
 end
+
+def self.refresh_concurrently
```

Comment:
I considered that, but in the end I decided that the `if` statement that would basically switch concurrent bit on or off wasn't as neat and tidy as two separate methods. Also, since it's possible for an error to be thrown if they refresh concurrently without a unique index, I figured that the stack trace would be nicer if they saw `refresh_concurrently` instead of just something going wrong with `refresh`.